webcomponents/tool/src/main/frontend/packages/sakai-grader/src/SakaiGrader.js

```diff
@@ -333,21 +333,17 @@ export class SakaiGrader extends graderRenderingMixin(gradableDataMixin(SakaiEle
     this._selectedAttachment = this._submission.submittedAttachments.find(sa => sa.ref === e.target.dataset.ref);
     const type = this._selectedAttachment.type;
 
-    if (type === "text/html") {
-      this._submittedTextMode = true;
-    } else {
-      this._submittedTextMode = false;
-      this.previewMode = true;
-      let preview = this._submission.previewableAttachments[this._selectedAttachment.ref];
-      preview = !preview && (type.startsWith("image/") || type.startsWith("video/") || this.previewMimetypes.includes(type)) ? this._selectedAttachment : preview;
+    this._submittedTextMode = false;
```

Comment:
I am not sure what this does, can this always be false?

Reply:
It triggers the display of the student's submitted text, ie their submission from the text area.

Reply:
I can help with the backport, if needed.

Reply:
Thanks, there are no hurries with the back-port. We are applying the initial fix for now, with small adjustments.